app/views.py

```diff
@@ -86,15 +94,17 @@ def createclass():
     """Renders the create class page for educators."""
     form = CreateClass()
     if form.validate_on_submit():
-        Group.name = form.className.data
+        group = Group(name=form.className.data)
+        db.session.add(group)
+        classCode = set_code(6)
+        db.session.add(classCode)
```

Comment:
```suggestion
        classCode = set_code(6)
        group = Group(name=form.className.data, classCode=classCode)
        db.session.add(group)
```
`classCode` is a `Column` in the `Group Table`. The while loop is also not done yet.

Reply:
classCode=classCode can be confusing, so I am going to put temp = set_code(6) then inside the group, classCode=temp. Missed the while loop for this page.